rq/job.py

```diff
@@ -733,4 +732,38 @@ def register_dependency(self, pipeline=None):
             connection.sadd(dependents_key, self.id)
             connection.sadd(self.dependencies_key, dependency_id)
 
+    def dependencies_finished(
```

Comment:
Is `job.dependencies_are_met` a better name? There could be cases where one of the dependencies have finished and are no longer in Redis becase it's cleaned up right away.

Reply:
Yes much appreciated - with https://github.com/rq/rq/pull/1170#discussion_r363012688, I see now that I have not been correctly considering jobs with `ttl=0`.

Reply:
changed to `dependencies_are_met` in 406d06106744c91cd945509c71a5e8bceac730fa